src/core_modules/capture-core/components/Pages/Search/SearchResults/SearchResults.types.js

```diff
@@ -25,7 +25,7 @@ export type PropsFromRedux ={|
 
 export type OwnProps ={|
     fallbackTriggered: boolean,
-    availableSearchOptions: AvailableSearchOptions,
+    availableSearchOptions: ?AvailableSearchOption,
```

Comment:
Same suggestion here about removing `s` from the end of the variable, just like in [SearchPage.types.js ](https://github.com/dhis2/capture-app/pull/3159/files#diff-9dc06406f1668010a7ea187da988e2d9c7e20caa892274965decb1ea72a36831R27)